README.md

```diff
@@ -1,88 +1,145 @@
-  ```bash
-  sudo apt update
-  sudo apt install tesseract-ocr python-opencv
-  ```
-2) Create a .csv with coords of gyms (one coord per line in format 'lat,lng'):
-  ```bash
-  lat1,lng1
-  lat2,lng2
-  ...etc
-  ```
-  You can also try to download the coords of gyms from your existing RM DB (downloadDBCords.py)
-3) Insert gym images into gym_img folder. Either manually (e.g. from Ingress) or
-  try the downloadfortimg.py
-4) Configure config.ini as needed. See `config.ini.example` for params and info
-5) Start TRM by calling `python startWalker.py`
-
-For performance it might be worth giving ocr_multitask a chance and/or calling
-startWalker.py with `-os` in one console and `-oo` in another.
-
-
-# Steps on Android device
-1) Install Pogo, droidVNC, RemoteGpsController, systemizer
-2) Systemize RGC (Oreo requires it to be systemized as a priv-app)
-3) Start VNC as noted above, start RGC
-
-# WE DO NOT GUARANTEE FOR THE ENTIRE THING TO BE RUNNING PERFECTLY FINE
-
-### Todos
-
- - Write tests
- - Write VNC app
- - Improve scans
- - Improve errorhandling
- - Support more/all resolutions
-
-# Discord
-For minor help, reporting resolutions (instructions on how to do so will be given sometime in the future), bugreports
-[Join the discord server](https://discord.gg/MC3vAH9)
+# Map'A'Droid
+![Python 2.7](https://img.shields.io/badge/python-2.7-blue.svg)  
+![MAD-Banner](https://raw.githubusercontent.com/OfficialWiddin/TheRaidMapper/patch-1/examples/banner_small_web.png)
+
+The Raid Mapper is a Raid scanner for Pokemon GO, based on Android devices and OCR.  
+
+<img src="https://raw.githubusercontent.com/Grennith/TheRaidMapper/master/examples/example_map.PNG" width="500" height="500">
+
+## Information
+*  [Discord](https://discord.gg/MC3vAH9) - For general support  
+*  [Github Issues](https://github.com/Grennith/TheRaidMapper/issues) - For reporting bugs (not for support!)  
+
+## Requirements  
+* [PogoAssets](https://github.com/ZeChrales/PogoAssets)  
+* Google or PTC account with minimum level 5 and no Raid-bosses in the dex  
+* Mobile with Android 8.0 (API 26) or higher
+* Root privileges [Magisk](https://forum.xda-developers.com/apps/magisk/official-magisk-v7-universal-systemless-t3473445) and [Terminal APP Systemizer](https://forum.xda-developers.com/apps/magisk/module-terminal-app-systemizer-ui-t3585851)  
+*  [Droid VNC Server](https://github.com/Grennith/TheRaidMapper/blob/master/APKs/VNC_app.apk) and [Remote GPS Controller](https://github.com/Grennith/TheRaidMapper/blob/master/APKs/RemoteGpsController.apk)  
 
+## Current limitations
+* The only supported aspect ratio is 16:9. If you have a softkey bar, disable it for PoGO.  
+* It takes time to load when you teleport between different locations. So faster phones may handle the loading better. We are testing on low end specs like [Redmi 5A](https://www.mi.com/in/redmi-5a/) for $75. A parameter to adjust the delays in between teleports and screenshots will most likely be added.   
```

Comment:
We should probably note, that the Redmi 5A is a bitch to setup with safetynet :p

Reply:
Maybe list devices some we know it is running on like (not an extensive list)
- Samsung S5 with Lineage 15.1
- Redmi 5 A....
- etc..

Reply:
- Motorola Moto G4

Reply:
HTC One M7 Lineage 14.1

Reply:
Samsung XCover 4 (Android 7 - Stock)